app/data_model/models.py

```diff
@@ -40,22 +42,25 @@ class EQSession(db.Model):
     user_id = db.Column('user_id', db.String, primary_key=True)
     session_data = db.Column('session_data', db.String)
     created_at = db.Column('created_at', db.DateTime, default=datetime.datetime.utcnow)
-    updated_at = db.Column('updated_at', db.DateTime, default=datetime.datetime.utcnow, onupdate=datetime.datetime.utcnow)
+    updated_at = db.Column('updated_at', db.DateTime)
```

Comment:
How does the `updated_at` get updated now?

Reply:
In __init__()

Reply:
so we set updated when we load from the DB?

Reply:
Indeed. I was a little skeptical too.

Reply:
Trying to keep it similar, but I can also do `on_update` since `expires_at` would be updated.